__tests__/Unit/Components/Navbar/Navbar.test.tsx

```diff
@@ -91,4 +93,22 @@ describe('Navbar', () => {
         const logo = getByTestId('logo');
         expect(logo).toBeInTheDocument();
     });
+
+    test('show login in button when the user is not logged in', () => {
+        server.use(
+            selfHandlerFn(404, {
+                message: 'User does not exist',
+                error: 'Not Found',
+            })
+        );
+        renderWithProviders(<NavBar />);
+        expect(getSignInNode('button')).toBeVisible();
+    });
```

Comment:
@RitikJaiswal75 this is the test that checks if the sign in button is showing or not, the tests should fail if the login button not showing